pysages/methods/umbrella_integration.py

```diff
@@ -5,11 +5,11 @@
 """
 Umbrella Integration.
 
-Umbrella integration uses multiple replica placed along a pathway in the free energy
+Umbrella integration uses multiple replicas placed along a pathway in the free energy
 landscape by Harmonic Bias simulations.
 From the statistics of the simulations, the thermodynamic forces along the path are
-determine and integrated to obtain an approximation of the free-energy landscape.
-This class implements the replica simulations and approximates the the free energy.
+determined and integrated to obtain an approximation of the free energy landscape.
+This class implements the replica simulations and approximates the free energy.
 However, the method is not very and accurate it is preferred that more advanced methods
 are used for the analysis of the simulations.
```

Comment:
Are these 2 lines needed? For some systems US may be enough. And eventually, it's up to the user make a wise choice.

Reply:
Yes, umbrella integration really isn't that great.
Especially with the approximations that we are making here.
(First order approximation instead of second order.)
We are planning to add better analysis methods later on. For now this should warn users not to expect great results.

Reply:
Right, right, now I got it. Maybe we could add "more advanced methods (e.g. the Weighted Histogram Analysis Method)". I've added this last bit with the latest commit (6e76c1d651812504aa8a9aad7f01b8d0201d177d). Let me know if you'd like to remove it at this point and I'll amend, otherwise go ahead and merge.